telemetry/telegraf/convert.go

```diff
@@ -218,7 +218,17 @@ func (s *metricFrame) append(m influx.Metric) error {
 			return err
 		}
 		if index, ok := s.fieldCache[f.Key]; ok {
-			s.fields[index].Append(v)
+			field := s.fields[index]
+			if ft != field.Type() {
+				fmt.Printf("error appending value of type: %v as %v // %v\n", field.Type(), ft, v)
```

Comment:
@FZambia what is the best logging option here?

Reply:
Maybe pass logger from the outside?